lang/po/Firmware_nl.po

```diff
@@ -0,0 +1,1787 @@
+# Translation of Prusa-Firmware into Dutch.
+#
+msgid ""
+msgstr ""
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: nl\n"
+"Project-Id-Version: Prusa-Firmware\n"
+"POT-Creation-Date: Thu, Oct 3, 2019 7:24:02 PM\n"
+"PO-Revision-Date: Thu, Oct 3, 2019 7:24:02 PM\n"
+"Language-Team: \n"
+"X-Generator: Poedit 2.0.7\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+"Last-Translator: \n"
+"Plural-Forms: nplurals=3; plural=(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2;\n"
+
+# 
+#: 
+msgid "[%.7s]Live adj. Z\x0avalue set, continue\x0aor start from zero?\x0a%cContinue%cReset"
+msgstr "[%.7s]Z instell.\x0awaarde ingesteld,\x0averder of bij 0\x0abeginen?\x0a%cVerder%cReset"
+
+# MSG_IMPROVE_BED_OFFSET_AND_SKEW_LINE2 c=14
+#: messages.c:39
+msgid " of 4"
+msgstr " van 4"
+
+# MSG_MEASURE_BED_REFERENCE_HEIGHT_LINE2 c=14
+#: messages.c:60
+msgid " of 9"
+msgstr " van 9"
+
+# MSG_MEASURED_OFFSET
+#: ultralcd.cpp:3082
+msgid "[0;0] point offset"
+msgstr "[0;0] punt offset"
+
+# MSG_CRASH_DET_ONLY_IN_NORMAL c=20 r=4
+#: 
+msgid "Crash detection can\x0abe turned on only in\x0aNormal mode"
+msgstr "Crashdetectie kan\x0aalleen in normaal\x0agebruikt worden"
+
+# MSG_CRASH_DET_STEALTH_FORCE_OFF c=20 r=4
+#: 
+msgid "WARNING:\x0aCrash detection\x0adisabled in\x0aStealth mode"
+msgstr "WARNING:\x0aCrashdetectie\x0auitgeschakeld in\x0aStealth stand"
+
+# 
+#: ultralcd.cpp:2470
+msgid ">Cancel"
+msgstr ">Annuleren"
+
+# MSG_BABYSTEPPING_Z c=15
+#: ultralcd.cpp:3202
+msgid "Adjusting Z:"
+msgstr "Z is ingesteld:"
+
+# MSG_SELFTEST_CHECK_ALLCORRECT c=20
+#: ultralcd.cpp:8402
+msgid "All correct      "
+msgstr "Allemaal goed    "
+
+# MSG_WIZARD_DONE c=20 r=8
+#: messages.c:101
+msgid "All is done. Happy printing!"
+msgstr "Klaar. Happy printing!"
+
+# 
+#: ultralcd.cpp:1993
+msgid "Ambient"
+msgstr "EINSY  "
+
+# MSG_PRESS c=20
+#: ultralcd.cpp:2619
+msgid "and press the knob"
+msgstr "en druk op knop"
+
+# MSG_CONFIRM_CARRIAGE_AT_THE_TOP c=20 r=2
+#: ultralcd.cpp:3521
+msgid "Are left and right Z~carriages all up?"
+msgstr "Zijn beide Z wagen heelemaal boven?"
+
+# MSG_AUTO_DEPLETE_ON c=17 r=1
+#: ultralcd.cpp:5267
+msgid "SpoolJoin    [on]"
+msgstr "SpoolJoin   [aan]"
+
+# 
+#: ultralcd.cpp:5263
+msgid "SpoolJoin   [N/A]"
+msgstr "SpoolJoin   [N/V]"
+
+# MSG_AUTO_DEPLETE_OFF c=17 r=1
+#: ultralcd.cpp:5271
+msgid "SpoolJoin   [off]"
+msgstr "SpoolJoin   [uit]"
+
+# MSG_AUTO_HOME
+#: messages.c:11
+msgid "Auto home"
+msgstr "Startpositie"
+
+# MSG_AUTOLOAD_FILAMENT c=17
+#: ultralcd.cpp:6928
+msgid "AutoLoad filament"
+msgstr "Auto-Laden Filament"
+
+# MSG_AUTOLOADING_ONLY_IF_FSENS_ON c=20 r=4
+#: ultralcd.cpp:4459
+msgid "Autoloading filament available only when filament sensor is turned on..."
+msgstr "Autoloading filament alleen beschikbaar wanneer filamentsensor is ingeschakeld ..."
```

Comment:
"Auto-laden" for consistency with the previous?

Reply:
"Auto. laden van fil. is enkel beschikbaar wanneer fil.sensor is ingeschakeld ..."
This fits perfectly the 20x4